### PR TITLE
[#2015] Fixed font usage in plugins

### PR DESCRIPTION
**Pull request by mjschranz**
https://webmademovies.lighthouseapp.com/projects/65733/tickets/2015-some-fonts-not-working-in-googlefonts-module

### CODE REVIEW

File: templates/assets/plugins/popup/popcorn.popup.js [hunk 1]
```diff
@@ -438,7 +439,7 @@
       fontSheet = document.createElement( "link" );
       fontSheet.rel = "stylesheet";
       fontSheet.type = "text/css";
-      options.fontFamily = options.fontFamily ? options.fontFamily.replace( /\s/g, "+" ) : "Tangerine";
```
Comment (jbuck) [inline templates/assets/plugins/popup/popcorn.popup.js:442]: Damn vegetables

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @jbuck received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/980c71080837045/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @jbuck received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/40ac47d9869cb5f/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/980c71080837045/output.txt

Total script time: 1.40 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/40ac47d9869cb5f/output.txt

Total script time: 1.08 mins

Server is running on http://pr805.butyr.org/templates/basic/
Run the Test Suite: http://pr805.butyr.org/test/

**Pull request merged**